app/src/generate.py

```diff
@@ -51,34 +48,27 @@ def get_models() -> dict[str, str]:
 def generate(
     llm: str,
     query: str,
-    context: Sequence[Chunk] | None = None,
+    context_text: str | None = None,
 ) -> str:
     """
     Returns a string response from an LLM model, based on a query input.
     """
+    messages = [
+        {
+            "content": PROMPT,
+            "role": "system",
+        }
+    ]
 
-    if context:
-        context_text = create_prompt_context(context)
-        messages = [
-            {
-                "content": PROMPT,
-                "role": "system",
-            },
+    if context_text:
```

Comment:
Minor refactoring to differentiate the returned value when `context_text` is present.